src/components/application_manager/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) 2016, Ford Motor Company
```

Comment:
Comma-separated would probably be better
```
Copyright (c) 2017, Ford Motor Company, Xevo Inc.
```

Reply:
```
- # Copyright (c) 2016, Ford Motor Company
+ # Copyright (c) 2017, Ford Motor Company, Xevo Inc.
```
In this case, the first year 2016 will disappear. Is it a policy of copyright notation of this project?

Reply:
I believe it would be fine since this file was updated by Ford/Luxoft this year, but since checking that requires time, I suppose we can just leave it as is.